### ISSUE TITLE
Keyboard input is not recognized at all

### ISSUE THREAD

**Issue by metalefty** [jesseduffield/lazygit#563]
**Describe the bug**
Keyboard input is not recognized at all

**To Reproduce**
1. git clone the repository (using commit dab78c8a632fa12a24db2247a5c7096c5166c91a)
2. `go build main.go`
3. `./main`
4. lazygit starts but doesn't recognize keyboard input

**Expected behavior**
Keyboard input is recognized and it works.

**Screenshots**
![スクリーンショット 2019-12-13 16 22 36](https://user-images.githubusercontent.com/941609/70778084-c7bbf180-1dc4-11ea-9215-c5b65af71e71.png)

**Desktop (please complete the following information):**
 - OS: FreeBSD 12
 - Lazygit dab78c8a632fa12a24db2247a5c7096c5166c91a

**Additional context**

**Comment by mjarkk** [comment]
Does this also happen on older lazygit versions for example [v0.7](https://github.com/jesseduffield/lazygit/releases/tag/v0.7)  
If it also doesn't work there you might be the first person to use lazygit on FreeBSD :)

**Comment by metalefty** [comment]
It didn't happen in older version when v0.7 was the latest version. It worked before.
Probably between v0.8..v0.9 the issue appeared first.

I'm trying to build lazygit v0.7 again, it fails to build for some reasons.
```
% go build main.go                                                                     git:(tags/v0.7)
# github.com/jesseduffield/lazygit/pkg/i18n
pkg/i18n/i18n.go:90:29: unknown field 'DefaultLanguage' in struct literal of type i18n.Bundle
```

I assume the issue is caused version change of requisite libraries but not sure.

**Comment by mjarkk** [comment]
Ya we used the vendor folder back then,  
you can build the project using the vendor folder this way:
```sh
export GO111MODULE=off
go build
```

If you don't have a working gopath you can try this inside the project:  
```sh
go get github.com/nicksnyder/go-i18n/v2@v2.0.0-beta.5
```

**Comment by metalefty** [comment]
```
export GO111MODULE=off
go build
```

This worked, thanks! Unfortunately, the built v0.7 still doesn't recognize keyboard inputs.
I don't remember but I might be using older go when v0.7 was the latest version.

```
go version go1.13.5 freebsd/amd64
```

**Comment by mjarkk** [comment]
There is also a FreeBSD download in the release that's definitely not build using a go version higher than 1.12.  
If that works maybe we should update the deps to try if that fixes your problem.

**Comment by dawidd6** [comment]
We are building releases using Go 1.13.

https://github.com/jesseduffield/lazygit/blob/dab78c8a632fa12a24db2247a5c7096c5166c91a/.circleci/config.yml#L39-L41

**Comment by mjarkk** [comment]
Sorry i meant the 0.7 version,  
https://github.com/jesseduffield/lazygit/blob/ac5088eee691de5285ab8e08d825ad398c85441c/.circleci/config.yml#L52

**Comment by miguelmota** [comment]
Add the following fix to the termbox-go fork that lazygit is using and it'll fix the keybinding issue on freebsd

https://github.com/nsf/termbox-go/pull/219/files#diff-651a84f8ad6a38c1f7ccdf63f2692410R30

**Comment by metalefty** [comment]
@mjarkk I'm not familiar with go build system, can you update your fork of termbox-go to @miguelmota pointed? I'm happy to test.

**Comment by jesseduffield** [comment]
I've just applied that change here https://github.com/jesseduffield/lazygit/pull/615. Should be good in the next release :)

**Comment by metalefty** [comment]
@jesseduffield Confirmed that the issue is fixed at v0.14!

**Comment by jesseduffield** [comment]
good to hear :)
